examples/camera/camera_coloured_ball_detect.py

```diff
@@ -5,18 +5,21 @@
 
 
 def process_frame(frame):
-    balls = ball_detector.detect(frame, color=("red", "green", "blue"))
+    detected_balls = ball_detector.detect(frame, color=("red", "green", "blue"))
 
-    red_ball = balls.red
-    print(f'Red ball center: {red_ball.center}')
+    red_ball = detected_balls.red
+    if red_ball.is_valid():
+        print(f'Red ball center: {red_ball.center}')
 
-    green_ball = balls.green
-    print(f'Green ball center: {green_ball.center}')
+    green_ball = detected_balls.green
+    if green_ball.is_valid():
```

Comment:
I prefer `red_ball.found` as it's more readable to a beginner when reading an if statement - thoughts?